src/core_plugins/kibana/public/discover/index.html

```diff
@@ -1,4 +1,4 @@
-<div ng-controller="discover" class="app-container">
+<div discover-app class="app-container">
```

Comment:
This can then become `<discover-app class="app-container">` (don't forget to change the closing tag to match).